pkg/fetch/metrics_fetcher.go

```diff
@@ -23,9 +23,9 @@ var (
 			Subsystem: "remote_asset",
 			Name:      "http_fetcher_blob_size_bytes",
 			Help:      "Size of blobs fetched using the http fetcher, in bytes",
-			Buckets:   util.DecimalExponentialBuckets(-3, 6, 2),
+			Buckets:   prometheus.ExponentialBuckets(1.0, 2.0, 33),
```

Comment:
Any reason for switching from the util implementation?

Reply:
This is how `size_bytes` fields are done in bb-storage so I used that as a guide. The util implementation uses powers of 10 rather than powers of 2 which seem to make more sense for numbers of bytes.